File: app/src/main/java/com/example/alkewalletapp/fragmentos/HomePage.java
package com.example.alkewalletapp.fragmentos;

import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import androidx.navigation.Navigation;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;
import android.widget.TextView;

import com.example.alkewalletapp.R;

public class HomePage extends Fragment {


    public HomePage() {
        // Required empty public constructor
    }


    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_home_page, container, false);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);



        View botonEnviar1 = view.findViewById(R.id.botonVerde1);
        View botonIngresar1 = view.findViewById(R.id.boton_ingresar1);


        botonEnviar1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Navigation.findNavController(v).navigate(R.id.action_homePage_to_sendMoney);

            }
        });

        botonIngresar1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Navigation.findNavController(v).navigate(R.id.action_homePage_to_requestMoney);
            }
        });
   }
}

File: app/src/main/java/com/example/alkewalletapp/fragmentos/HomePage.kt
package com.example.alkewalletapp.fragmentos

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.navigation.Navigation.findNavController
import com.example.alkewalletapp.R

class HomePage : Fragment() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_home_page, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)


        val botonEnviar1 = view.findViewById<View>(R.id.botonVerde1)
        val botonIngresar1 = view.findViewById<View>(R.id.boton_ingresar1)


        botonEnviar1.setOnClickListener { v -> findNavController(v).navigate(R.id.action_homePage_to_sendMoney) }

        botonIngresar1.setOnClickListener { v -> findNavController(v).navigate(R.id.action_homePage_to_requestMoney) }
    }
}